recursive_art.py

```diff
@@ -175,10 +180,53 @@ def generate_art(filename, x_size=350, y_size=350, frames=1):
                             color_map(blue_function(x, y))
                             )
         im.save(filename + str(frame) + ".png")
+        print("Frame" + str(frame) + " done.")
+
+
+def get_img_surfaces(direc="frames"):
+    """
+    Converts all images in direc to pygame surfaces
+    """
+    files = sorted(listdir(direc))  # List all files in directory
+    all_imgs = [pygame.image.load("frames/" + file_name) for file_name in files]  # Load the images using pygame
+    return all_imgs
+
+
+class AudioInput():
```

Comment:
Not positive if this wrapper is better than making an out-of-the-box alsaaudio.PCM in a normal function, but still a reasonable application of object-oriented programming.